tests/fig42-dff/README.md

```diff
@@ -0,0 +1,17 @@
+## DFF tests
```

Comment:
This test is already at https://github.com/SymbiFlow/python-symbiflow-v2x/tree/master/tests/vtr/dff